runtime/gcchk/gcchk.cpp

```diff
@@ -420,8 +420,10 @@ hookGcCycleEnd(J9HookInterface** hook, UDATA eventNum, void* eventData, void* us
 
 	UDATA oldVMState = vmThread->omrVMThread->vmState;
 	vmThread->omrVMThread->vmState = OMRVMSTATE_GC_CHECK_AFTER_GC;
+	/* the high bit(OMR_GC_CYCLE_TYPE_STATE_UNSUCCESSFUL) is set in the event->cycleType for cycleEnd in scavenge backout case, so mask off the bit to get the cycleType without the state. */
```

Comment:
- please do not mention "high bit" (it is not for 64-bit), just say "OMR_GC_CYCLE_TYPE_STATE_UNSUCCESSFUL bit is set ..."
- this comment is long enough, would it be better to use multi-line comment format?